river/datasets/synth/planes_2d.py

```diff
@@ -0,0 +1,73 @@
+import random
+
+from .. import base
+
+
+class Planes2D(base.SyntheticDataset):
+    """2D Planes synthetic dataset.
+
+    This dataset is described in [^1] and was adapted from [^2]. The features are generated
+    using the following probabilities:
+
+    $$P(x_1 = -1) = P(x_1 = 1) = \\frac{1}{2}$$
+
+    $$P(x_m = -1) = P(x_m = 0) = P(x_m = 1) = \\frac{1}{3}, m=2,\\ldots, 10$$
+
+    The target value is defined by using rule:
```

Comment:
Suggested rewording: "defined by the following rule"

Reply:
Reworded in commit [afa307d](https://github.com/online-ml/river/pull/387/commits/afa307da973ab1519b2d06ae3852ba566f5ee147).